pandas/core/indexes/accessors.py

```diff
@@ -200,13 +232,24 @@ class TimedeltaProperties(Properties):
     """
     Accessor object for datetimelike properties of the Series values.
 
-    Examples
-    --------
-    >>> s.dt.hours
```

Comment:
Forgot to mention, I could not manage to give an example for ```s.dt.hours```, 

I tried to have:

```python
import pandas as pd
hours_series = pd.Series(pd.timedelta_range(start="1 day", periods=3, freq="H"))
```

And then:

```python
hours_series.dt.hours
```

and

```python
 hours_series.dt.hour
```

but both gives:

```python-traceback
AttributeError: 'TimedeltaProperties' object has no attribute 'hour'
AttributeError: 'TimedeltaProperties' object has no attribute 'hours'
```

Reply:
can you see if there is an issue for this.

```
>>> s = pd.Series(pd.timedelta_range(start="1 day", periods=3, freq="H"))
>>> s
0   1 days 00:00:00
1   1 days 01:00:00
2   1 days 02:00:00
dtype: timedelta64[ns]
>>>
>>> s.dt.components
   days  hours  minutes  seconds  milliseconds  microseconds  nanoseconds
0     1      0        0        0             0             0            0
1     1      1        0        0             0             0            0
2     1      2        0        0             0             0            0
```

Reply:
I could not find an issue for this, so I opened one here:

xref #33255